### PR TITLE
chore: change instances of "qgb" to "Blobstream", update workflow

### LINKED ISSUE #1113: chore: change qgb -> blobstream
- [ ] rename pages
- [ ] rename workflow
- [ ] add redirects for old URIs

### LINKED ISSUE #1114: chore: remove celestium from documentation
- [ ] replace with ?
- [ ] update all diagrams with "celestium" in them

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

This PR resolves #1113 
- [x] update non-programmatic instances of "QGB" and "Quantum Gravity Bridge" to "Blobstream"
- [x] update programmatic instances of `qgb` to new name, once renamed
- [x] remove mentions of Celestiums and replace with "layer 2 (L2)" and "layer 2s (L2s)" Resolves #1114 
- [x] update workflow to pull in source from orchestrator-relayer repository with correct names, i.e. `blobstream-xxx.md`
- [x] add new diagrams draft
- [x] add final versions of new diagrams

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/community/cohort-one-content.md [hunk 1]
```diff
@@ -137,8 +137,8 @@ description: The content and calendar of the first Modular Fellows cohort.
 
 - **Monday 21 November @ 11am EST / 4pm UTC / 5pm CEST**
   - In this lesson, Evan Forbes will teach us about the Quantum Gravity Bridge
-    (QGB) and [Celestiums](https://blog.celestia.org/celestiums). The QGB proves
-    that some data was posted on Celestia. Celestiums use the this bridge to
+    (Blobstream) and [layer 2s](https://blog.celestia.org/celestiums). The Blobstream proves
```
Comment (SweeXordious) [inline docs/community/cohort-one-content.md:140]:
```suggestion
    (Blobstream) and [layer 2s](https://blog.celestia.org/celestiums). Blobstream proves
```

File: docs/nodes/blobstream-binary.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    cd orchestrator-relayer<br />
+    git checkout {constants.orchrelayVersion}<br />
+    </code></pre>
+```
+
+3. Install the Blobstream CLI
+
+   ```sh
+   make install
+   ```
```
Comment (SweeXordious) [inline docs/nodes/blobstream-binary.mdx:32]:
```suggestion
```sh
make install
```
```

File: docs/nodes/blobstream-binary.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+# Run tests
+make test
+
+# Format code with linters (this assumes golangci-lint and markdownlint are installed)
+make fmt
+```
+
+## Useful links
+
+The smart contract implementation is in [blobstream](https://github.com/celestiaorg/blobstream/).
```
Comment (SweeXordious) [inline docs/nodes/blobstream-binary.mdx:77]:
```suggestion
The smart contract implementation is in [blobstream](https://github.com/celestiaorg/blobstream-contracts/).
```

File: docs/nodes/blobstream-intro.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Blobstream
+description: An overview of the Blobstream.
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:3]:
```suggestion
description: An overview of Blobstream.
```

File: docs/nodes/blobstream-intro.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+---
+
+# Blobstream
+
+The Blobstream, covered previously in
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:8]:
```suggestion
Blobstream, covered previously in
```

File: docs/nodes/blobstream-intro.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+[the Celestia blog post](https://blog.celestia.org/celestiums/),
+introduces the concept of a layer 2 (L2), which is an EVM L2
+rollup that uses Celestia for data availability
+but settles on Ethereum (or any other EVM L1 chain).
+
+This page and following tutorials will go over the Quantum Gravity
+Bridge and how Validators on Celestia can run it.
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:15]:
```suggestion
This page and following tutorials will go over Blobstream and how Validators on Celestia can run it.
```

File: docs/nodes/blobstream-intro.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+If you're looking to learn more, you can view
+[the `orchestrator-relayer` repository](https://github.com/celestiaorg/orchestrator-relayer)
+, and
+[read more about the Blobstream](https://github.com/celestiaorg/quantum-gravity-bridge#how-it-works).
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:20]:
```suggestion
[read more about Blobstream](https://github.com/celestiaorg/quantum-gravity-bridge#how-it-works).
```

File: docs/nodes/blobstream-intro.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+## Overview
+
+The Blobstream,
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:24]:
```suggestion
Blobstream,
```

File: docs/nodes/blobstream-intro.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+consists of two components: an [Orchestrator](../blobstream-orchestrator)
+and a [Relayer](../blobstream-relayer).
+
+In the following diagram, we show how a layer 2 would post the data to
+Celestia. This will later be attested to by the Celestia validator set, and
+eventually posted to the target EVM chain (in this case, Ethereum). Then,
+the layer 2, or any party, will be able to verify the attestations, i.e. valsets
+and data commitments, directly on the EVM chain on the Blobstream smart contract. You can
+reference
+[the Blobstream smart contract](https://github.com/celestiaorg/quantum-gravity-bridge/blob/master/src/QuantumGravityBridge.sol).
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:34]:
```suggestion
[the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
```

File: docs/nodes/blobstream-intro.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+
+![Blobstream-Architecture](../img/blobstream/blobstream-diagram.png)
+
+The specification of the Blobstream `Valset`s, which track the Celestia validator set
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:38]:
```suggestion
The specification of Blobstream `Valset`s, which track the Celestia validator set
```

File: docs/nodes/blobstream-intro.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+changes, can be found in [ADR 002](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-002-qgb-valset.md).
+
+The Blobstream data commitments, which represent commitments over sets of blocks
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:41]:
```suggestion
Blobstream data commitments, which represent commitments over sets of blocks
```

File: docs/nodes/blobstream-intro.md [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+but Blobstream offers a more decentralized, secure, and scalable solution
+compared to the potential centralized nature of DACs.
+
+## Setting up the Blobstream
+
+The following sections in this category presume you have the following setup:
+
+- A Celestia App
+[validator node](../consensus-node#optional-setting-up-a-validator) running
+- A Celestia Node [bridge node](../bridge-node) running
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:104]: this one is not needed for running Blostream

File: docs/nodes/blobstream-intro.md
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:1]: I will make another pass on this after the other feedback is addressed

File: docs/nodes/consensus-node.mdx [hunk 13]
```diff
@@ -586,12 +586,12 @@ txhash: <tx-hash>
 You should now be able to see your validator from
 [a block explorer](../mocha-testnet#explorers)
 
-#### Run the QGB Orchestrator
+#### Run the Blobstream Orchestrator
 
-Now that the QGB will be enabled for Mocha, all validators
-will need to run the QGB orchestrator to be able to sign attestations.
+Now that the Blobstream will be enabled for Mocha, all validators
```
Comment (SweeXordious) [inline docs/nodes/consensus-node.mdx:591]:
```suggestion
Now that Blobstream will be enabled for Mocha, all validators
```

**Review by SweeXordious — COMMENTED** [review]
First pass looks good :+1: :+1: 

Left some cosmetic comments to be addressed. Then, I will make a second pass on the overview

File: docs/nodes/blobstream-binary.mdx (hunk 2 above)
Comment (jcstein) [reply docs/nodes/blobstream-binary.mdx:32]: I'm going to resolve this without committing suggestion, as the indentation was done by Prettier and[ formats the numbered list properly in the preview](https://docs.celestia.org/pr-preview/pr-1115/nodes/blobstream-binary/)

File: docs/nodes/blobstream-intro.md (hunk 12 above)
Comment (jcstein) [reply docs/nodes/blobstream-intro.md]: 1106558898e8cf71bd448d621c7ec215f351e4c1 removed bridge node

File: docs/nodes/blobstream-binary.mdx (hunk 2 above)
Comment (SweeXordious) [reply docs/nodes/blobstream-binary.mdx:32]: cool :+1:

File: docs/nodes/blobstream-intro.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+Blobstream is built on Celestia, which uses a CometBFT-based proof-of-stake
+system. An incorrect data availability attestation in this system can be
+penalized, ensuring validators act in good faith. In contrast, data availability
+committees (DACs), are typically centralized or semi-centralized, relying on
+a specific set of entities or individuals to vouch for data avaiability.
+
+### Mechanism of verification
+
+Blobstream uses data availability attestations, which are Merkle roots of the
+L2 data, to confirm that the necessary data is present on Celestia. When
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:80]:
```suggestion
batched L2 data, to confirm that the necessary data is present on Celestia. When
```

File: docs/nodes/blobstream-intro.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+Ethereum's L2 contract updates its state, it checks with Blobstream to confirm
+that data's presence on Celestia. In contrast, a DAC would rely on attestations
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:82]: > it checks with Blobstream to confirm that data's presence on Celestia

How this? I am not aware of this happening or I don't get this part

File: docs/nodes/blobstream-intro.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+or confirmations from its permissioned members.
+
+### Flexibility and scalability
+
+Blobstream offers high-throughput data availability for Ethereum L2s, striking
+a balance between scalability and security. It isn't bound by Ethereum's gas
+costs, as Celestia's resource pricing is more byte-focused rather than
+computation-centric. The scalability costs of a DAC would largely hinge on its
+specific design, but it may not achieve the same balance as Blobstream.
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:91]: Not sure about this too

File: docs/nodes/blobstream-intro.md (hunk 15 above)
Comment (jcstein) [reply docs/nodes/blobstream-intro.md]: reworked in https://github.com/celestiaorg/docs/pull/1115/commits/ba3b63238fa88379334c509b748d14ec8ad42f91

File: docs/nodes/blobstream-intro.md [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+
+In the following diagram, we show how a layer 2 would post data to
+Celestia and then verify that it was published in the target EVM chain.
+
+In fact, data will first be attested to by the Celestia validator set, i.e. signing commitments committing to the data. Then, these signatures will be relayed to the target EVM chain (in this case, Ethereum). Finally,
+the layer 2, or any party, will be able to verify that the data was published to Celestia directly on the EVM chain on the Blobstream smart contract. You can
+reference
+[the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
+
+![Blobstream-Architecture](../img/blobstream/Blobstream.png)
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:35]:
```suggestion
In the following diagram, we show how a layer 2 would post data to
Celestia and then verify that it was published in the target EVM chain.

![Blobstream-Architecture](../img/blobstream/Blobstream.png)

In fact, data will first be attested to by the Celestia validator set, i.e. signing commitments committing to the data. Then, these signatures will be relayed to the target EVM chain (in this case, Ethereum). Finally,
the layer 2, or any party, will be able to verify that the data was published to Celestia directly on the EVM chain on the Blobstream smart contract. You can
reference
[the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
```

File: docs/nodes/blobstream-intro.md [hunk 18]
```diff
@@ -0,0 +1,110 @@
...
+- The Orchestrator: queries the attestation, signs it, then submits the signature
+  to the Blobstream P2P network
+
+The diagram below goes over this process.
+
+![Blobstream-Orchestrator](../img/blobstream/blobstream-orchestrator.png)
+
+The relayer submits the attestations' signatures from the Blobstream P2P network to the target EVM chain.
+
+> **_NOTE:_** If the contract is still not deployed, then it needs to be deployed before its used by the relayer.
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:58]:
```suggestion
> **_NOTE:_** If the contract is still not deployed, then it needs to be deployed before its used by the relayer. Check the [deployment documentation](./blobstream-deploy.md) for more details.
```

File: docs/nodes/blobstream-intro.md [hunk 19]
```diff
@@ -0,0 +1,110 @@
...
+You can learn more about the mechanics behind the Relayer in
+[ADR 004](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-004-qgb-relayer-security.md).
+
+## Blobstream and data availability committees (DACs)
+
+### Decentralization and security
+
+Blobstream is built on Celestia, which uses a CometBFT-based proof-of-stake
+system. An incorrect data availability attestation in this system can be
+penalized, ensuring validators act in good faith. Thus, Blobstream shares the same security assumptions as Celestia. In contrast, data availability
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:73]:
```suggestion
system. An incorrect data availability attestation in this system will ultimately be
penalized (currently not implemented), ensuring validators act in good faith. Thus, Blobstream shares the same security assumptions as Celestia. In contrast, data availability
```

File: docs/nodes/blobstream-intro.md [hunk 20]
```diff
@@ -0,0 +1,110 @@
...
+committees (DACs), are typically centralized or semi-centralized, relying on
+a specific set of entities or individuals to vouch for data avaiability.
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:75]:
```suggestion
a specific set of entities or individuals to vouch for data availability.
```

File: docs/nodes/blobstream-intro.md [hunk 21]
```diff
@@ -0,0 +1,110 @@
...
+
+### Mechanism of verification
+
+Blobstream uses data availability attestations, which are Merkle roots of the batched
+L2 data, to confirm that the necessary data is present on Celestia. The L2 contract
+on Ethereum doesn't directly check with Blobstream, but relies on the data
+availability guarantee provided by Blobstream. In contrast, a DAC would rely on
```
Comment (SweeXordious) [inline docs/nodes/blobstream-intro.md:82]:
```suggestion
on Ethereum can check directly with Blobstream if the data is published on Celestia. Similarly, a DAC would rely on
```

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**